### PR TITLE
Saving and loading bought coins to file

### PR DESCRIPTION
**Pull request by Rybens92**
Saving them to file, so program doesn't 'forget' previous actions between restarts.

I don't know if is it really needed feature, because my understandings of python code isn't that well, but maybe it might be helpful

### CODE REVIEW

File: news-analysis.py [hunk 1]
```diff
@@ -440,14 +446,36 @@ def sell(compiled_sentiment, headlines_analysed):
...
+def load_coins_in_hand_from_file():
+    path = 'coins_in_hand.dat'
+
+    # abort if file doesn't exist
+    if not os.path.isfile(path):
+        return
+
+    with open(path, 'rb') as file:
+        coins_in_hand = pickle.load(file)
+        file.close()
```
[the remark this answers is not in the document]
Comment (Rybens92) [reply news-analysis.py:468]: Yeah, you are right. After looking online when to use close function, it turns out that using 'with' automaticly closes a file at the end of it. Thanks for a hint.

**Comment by Rybens92** [comment]
Have made little mess with this branch 😅 but I hope it's all readable

I thought that saving to json file would be better, because of user's possibility to edit it even with a normal notepad 

What do you think @CyberPunkMetalHead ? Is it needed? Anything I can change?

**Comment by CyberPunkMetalHead** [comment]
Hey @Rybens92, I just had a quick look at history for this branch. Let me figure out what's going on here xD

**Comment by Rybens92** [comment]
😆
Sorry, maybe just close this branch and I will make another one without so much mess 😅

**Comment by CyberPunkMetalHead** [comment]
haha, no worries I'll close this one! 😆

**Pull request closed without merging**